arbitrum-docs/partials/_troubleshooting-users-partial.md

```diff
@@ -160,3 +160,23 @@ A week is expected to be more than enough time for validators to carry out an in
 
 
 
+### Are there any plans for reducing the time a transaction needs to wait before being able to be forced included from Ethereum into the Arbitrum chain, bypassing the sequencer? (Currently 24 hours) {#are-there-any-plans-for-reducing-the-time-a-transaction-needs-to-wait-before-being-able-to-be-forced-included-from-ethereum-into-the-arbitrum-chain-bypassing-the-sequencer-currently-24-hours}
+<p>The mechanism that allows force-including transactions from Ethereum (bypassing the sequencer) is intended to be used in very rare cases, specially when it is expected that the sequencer will not be operational again, so that users have a way of interacting with Arbitrum in a trustless way.</p>
```

Comment:
```suggestion
<p>The mechanism that allows force-including transactions from Ethereum (bypassing the sequencer) is intended to be used in very rare cases, specially when it is expected that the sequencer will not be operational again, so that users have a way of interacting with Arbitrum in a trustless way.</p>
```

Not blocking, but "specially" may need to be updated in Notion - assuming your intent was "especially".